Level-Editor.md

```diff
@@ -9,3 +9,58 @@ There are various videos on the internet documenting the use of the built-in lev
 - [Advanced Tutorial (Part 1/2)](https://www.youtube.com/watch?v=WBdwwcLD-vw) by Oliver Buo
 - [Advanced Tutorial (Part 2/2)](https://www.youtube.com/watch?v=UoaGDuBax6E) by Oliver Buo
 - [Expert Tutorial](https://www.youtube.com/watch?v=lL3oZbPfw08) by Oliver Buo
+
+## Share Levels
+
+When you finished a level or even more, there is an easy way to share it to the SuperTux community.
+
+### Step 1) Go to the SuperTux folder and check if all files are there.
+You can find them at C:\Users\YOUR-USERNAME\AppData\Roaming\SuperTux\supertux2\levels\YOUR-LEVELSETNAME\ on Windows or /home/YOUR-USERNAME/.local/share/supertux2/levels/YOUR-LEVELSETNAME/ on Linux.
```

Comment:
- Mark code-like stuff (that includes paths like these) with \` (backticks), `so that it looks like this`.
- Add path used on macOS

Reply:
I don't know the path used on macOS. I'm using Linux and I found the Windows path somewhere at the forum, but macOS...